test/helpers/rpc.js

```diff
@@ -0,0 +1,25 @@
+const pify = require("pify");
+
+/**
+ * Generic RPC interface
+ *
+ * @param {string} method Name of provider method call
+ * @param {Array} params Argument for method call
+ * @param {Object} web3 Web3 interface
+ */
+
+const rpcSend = (method = "", params = [], web3) => {
+  const send = pify(web3._provider.send.bind(web3._provider));
+  // an "id" is required here because the web3 websocket provider (v1.0.0-beta.35) throws if it is
+  // missing (it's probably just a bug on their end)
+  return send({
+    id: "1",
```

Comment:
An auto-incrementing id here would probably be better, I think.

The [JSON-PRC spec](https://www.jsonrpc.org/specification) intends for the `id` to be used to match requests and responses. Not sure how useful this is for us when using an http provider, but it would be useful in websocket communication.

Reply:
I'm not sure either, but can add auto-incrementing but not sure how to test this yet.